react/lifecycle-methods.md

```diff
@@ -7,6 +7,6 @@ Lifecycle in this context refers to the different stages of an instantiated comp
 The most important to start with are :
   - `componentWillMount` - before render
   - `componentDidMount` - after render (and has been mounted)
-  - `componentWillUnmount` - before umount
+  - `componentWillUnmount` - before unmount
```

Comment:
Haha ... this was probably me. Intuitively, the linux command to mount a drive is `mount`. Unintuitively, to unmount a drive, the command is `umount`. Now it's in my muscle memory to misspell unmount 😞 😆